setup.cfg

```diff
@@ -1,3 +1,24 @@
+[metadata]
+name = dahuffman
+author = Stefaan Lippens
+author_email = soxofaan@gmail.com
+license = MIT
+description = Pure Python Huffman encoder and decoder module
+keywords = huffman, compression, encoding, decoding
+url = https://github.com/soxofaan/dahuffman
+long_description = file: README.rst
+#long_description_content_type = 
```

Comment:
Any reason  long_description_content_type is commented? Is that still a TODO?

Reply:
I planned to fill it in but forgot.